client/src/components/HomePage/index.js

```diff
@@ -1,17 +1,90 @@
 import React, { Component } from "react";
-import { Link } from "react-router-dom";
+import Header from "../Header/header";
+import NavBar from "../NavBar";
+import axios from "axios";
 import "./HomePage.css";
-
+import { Link } from "react-router-dom";
 class HomePage extends Component {
+  state = {
+    loading: false
+  };
+
+  componentDidMount() {
+    axios
+      .get("/service")
+      .then(({ data }) => {
+        const services = data.map(service => {
+          return {
+            id: service.id,
+            image_url: service.image,
+            name: service.name
+          };
+        });
+        this.setState({
+          services,
+          loading: true
+        });
+      })
+      .catch(() => {
+        const { history } = this.props;
+        history.push("/error");
+      });
+  }
+
+  redirectToEventPage = hall => {
```

Comment:
Why did you write this ```redirectToEventPage ```function! you didn't use it!

Reply:
3a asas bt3rfesh y3ne :p 
I forgot to delete it :3